apps/plotly_examples/src/components/DeclarativeChart.tsx

```diff
@@ -49,7 +49,7 @@ type DataType =
   | 'plotly_express_colors';
 
 const dataTypeRanges = {
-  'general': [{ min: 1, max: 252 }],
+  'general': [{ min: 1, max: 252 }, {min: 750, max: 757 }],
```

Comment:
add these 8 entries in playwright.spec file also to pick it up in playwright tests